index.android.d.ts

```diff
@@ -131,30 +131,28 @@ declare module 'react-native-google-fit' {
       inLocalTimeZone: boolean
     ) => Promise<AggregatedHeartRateResponse[]>
 
-    getAggregatedHeartRateSamples: (
```

Comment:
I don't think you should be removing this (or the type export below) unless I'm missing something?

Reply:
This was a duplicate that is defined above, GitHub has cut it out of the changes